public/API/API_GetUserGameAwards.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ *  API_GetUserGameAwards - gets user's completion and mastery awards
+ *    u : username
+ *
+ *  array
+ *   object     [value]
+ *    string     AwardedAt                    iso 8601 date representing when the award was earned by the user
+ *    string     GameID                       unique identifier of the game
+ *    string     GameName                     name of the game
+ *    string     ConsoleID                    unique identifier of the console associated to the game
+ *    string     ConsoleName                  name of the console associated to the game
+ *    bool       IsMastery                    true if the award was earned while in hardcore mode
+ *    string     DisplayOrder                 used for determining which order to display the awards
+ *    bool       IsMissingAchievementUnlocks  true if new achievements were added to the set after the award was earned
+ */
+
+use LegacyApp\Community\Enums\AwardType;
+
+$user = request()->query('u');
+
+$allUserSiteAwards = getUsersSiteAwards($user);
+
+$filteredAwards = [];
+foreach ($allUserSiteAwards as $userSiteAward) {
+  $currentAwardType = (int) $userSiteAward['AwardType'];
```

Comment:
ℹ️ `.editorconfig` assumes 4 spaces for php files

Reply:
Updated